src/shared/components/common/markdown-textarea.tsx

```diff
@@ -150,6 +161,17 @@ export class MarkdownTextArea extends Component<
                 dangerouslySetInnerHTML={mdToHtml(this.state.content)}
               />
             )}
+            {this.state.imageUploadStatus &&
+              this.state.imageUploadStatus.total > 1 && (
+                <ProgressBar
+                  className="mt-2"
+                  striped
+                  animated
+                  value={this.state.imageUploadStatus.uploaded}
+                  max={this.state.imageUploadStatus.total}
+                  text={`${this.state.imageUploadStatus.uploaded}/${this.state.imageUploadStatus.total} files uploaded`}
```

Comment:
We'll have to add a translation for this.

Reply:
Make a PR over on lemmy-translations for me to merge. Here's an example of using counts: https://github.com/LemmyNet/lemmy-translations/blob/main/translations/en.json#L148

You can grep the lemmy-ui for any of those formatted count type strings to see how to do in the UI.